starboard/android/apk/app/src/main/java/dev/cobalt/media/ArtworkDownloader.java

```diff
@@ -0,0 +1,22 @@
+// Copyright 2024 The Cobalt Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package dev.cobalt.media;
```

Comment:
Just wanted to double check if we need to add this file to "apk_sources.gni".

Reply:
Yes, we need to add the new files in the gni file, thanks for pointing this out.